### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by gokhanbarisaker**

### CODE REVIEW

File: lib/src/test/java/io/ably/lib/test/util/TokenServer.java [hunk 1]
```diff
@@ -54,7 +56,7 @@ public Response serve(IHTTPSession session) {
         Map<String, String> headers = session.getHeaders();
 
 		if (!method.equals(Method.GET)) {
-			return new Response(Response.Status.METHOD_NOT_ALLOWED, MIME_PLAINTEXT, "Method not supported");
+			return newFixedLengthResponse(Response.Status.METHOD_NOT_ALLOWED, MIME_PLAINTEXT, "Method not supported");
```
Comment (mattheworiordan) [inline lib/src/test/java/io/ably/lib/test/util/TokenServer.java]: Out of interest why this change?

Comment (gokhanbarisaker) [inline lib/src/test/java/io/ably/lib/test/util/TokenServer.java]: New version does not allow direct access to `Response` constructor.

File: build.gradle [hunk 2]
```diff
@@ -11,7 +11,7 @@ allprojects {
   group = 'io.ably'
 
   repositories {
-      mavenCentral()
+      jcenter()
       maven { url "https://raw.github.com/paddybyers/Java-WebSocket/mvn-repo/" }
```
Comment (paddybyers) [inline build.gradle]: @gokhanbarisaker did you ever think of a way to avoid this?

**Comment by paddybyers** [comment]
LGTM.
